felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -212,6 +212,15 @@ var zeroIface bpfInterface = func() bpfInterface {
 	return i
 }()
 
+type bpfHostIface struct {
+	name        string
+	index       int
+	masterIndex int
+	parentIndex int
```

Comment:
can the interface have both master and parent in the tree? It feels to me that that the system ifaces can have either or, but not both and it is just a naming convention. I wonder if it would make thing simple if we maintained only a parent for both in our internal tree. Or maybe I am confused since there is just one parent pointer.